app/views/dashboard/_book.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="carousel-item<%= if book_iteration.first? then ' active' end %>">
+  <div class="row" style="height: 400px;">
+    <div class="col-12 text-center" style="height: 400px;">
+      <h1><%=book.name%></h1>
+      <% book_cover = book.cover.attached? ? book.cover : "sample-book-cover.jpg" %>
```

Comment:
This logic is now in two places - here and `app/views/books/_book.html.erb`

This is probably something we should think about wrapping up in a helper method so we don't need to change this logic in multiple places if the sample book cover name changed.

Essentially what I think we should end up with here is:

```ruby
<%= image_tag(your_method_here(book.cover), class: "p-2 h-100") %>
```

Where `your_method_here` internally does the logic found here: `book_cover = book.cover.attached? ? book.cover : "sample-book-cover.jpg" `

I'm going to leave the naming of `your_method_here` up to you.